js/src/components/paid-ads/ads-campaign.js

```diff
@@ -101,10 +108,16 @@ export default function AdsCampaign( {
 			<StepContentFooter>
 				<AppButton
 					isPrimary
-					disabled={ ! isValidForm }
+					disabled={
+						! isValidForm ||
+						( trackingContext === 'setup-ads' &&
+							billingStatus?.status !==
+								GOOGLE_ADS_BILLING_STATUS.APPROVED )
+					}
+					loading={ isLoading }
 					onClick={ onContinue }
 				>
-					{ __( 'Continue', 'google-listings-and-ads' ) }
+					{ submitButtonText }
```

Comment:
@eason9487 I believe this continue button is only temporarily going to need to handle this many conditions until #2535 is completed, to ensure that the current flow could still be tested properly. I'd like to avoid making any additional modifications to the `AdsCampaign` component in this PR, which is really only meant to remove the billing step because it will be handled during step 3 once we've consolidated the form from the onboarding flow.

Reply:
https://github.com/woocommerce/google-listings-and-ads/blob/34c53d7d4d37fbfdf46b00aa36195c2e09710eb4/js/src/components/paid-ads/ads-campaign/ads-campaign.js#L103-L117

https://github.com/woocommerce/google-listings-and-ads/blob/34c53d7d4d37fbfdf46b00aa36195c2e09710eb4/js/src/components/paid-ads/ads-campaign/ads-campaign.js#L176-L182

From the current revision of `AdsCampaign` in #2623, I couldn't figure out how this button only *temporarily* needs to handle 5 props, because both PRs currently increase the complexity and conditional branching of this button to varying degrees.

All of these changes convinced me that both PRs should try to see if it's possible to move this button out of the `AdsCampaign` component sooner rather than having to resolve more code conflicts and re-test all button scenarios when merging them with each other.